package/gluon-autoupdater/luasrc/lib/gluon/upgrade/500-autoupdater

```diff
@@ -44,7 +44,7 @@ autoupdater_util.randomseed()
 -- a fallback update (used after the regular updates haven't worked for
 -- (priority+1) days after a firmware release, for example because the node
 -- is always offline at night)
-local minute = math.random(0, 59)
+local minute = uci:get('autoupdater', 'settings', 'minute') or math.random(0, 59)
```

Comment:
This adds a Gluon-specific option ('minute') to a non-Gluon config file (the autoupdater is in the *packages* repo).

I would prefer to get https://github.com/freifunk-gluon/gluon/pull/1216/commits/594f45d7d1aad9364d67f65aaf3249a9067d10f3 (#1216) merged first, so we can call this `gluon.autoupdater.minute`.